ov/connectionv3.go

```diff
@@ -0,0 +1,63 @@
+/*
+(c) Copyright [2015] Hewlett Packard Enterprise Development LP
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package ov -
+package ov
+
+import (
+	"github.com/HewlettPackard/oneview-golang/utils"
+)
+
+func (c *OVClient) ManageI3SConnections(connections []Connection) []Connection {
+
+	if len(connections) < 4 {
```

Comment:
It is correct the way it is. There are 2 connections for our profile template. And we want to add 2 more...unless they are already there. I don't like it but this is temporary until we can figure out what we want to do